blocks/contact-form/contact-form.js

```diff
@@ -69,27 +75,51 @@ export default function decorate(block) {
       div.removeChild(inputTypeColumn);
     }
   });
+
   // Specify submitContactInfo arguments
   const submitInfo = async (event) => {
     // To prevent the button from scrolling up when clicked
     if (event) {
       event.preventDefault();
     }
 
-    // Make sure no inputs are empty
+    // Select empty inputs
     const emptyInputs = [...formInputs].filter(
```

Comment:
with this check, you're not actually getting the _empty_ inputs, but the invalid ones, right? for an email input, this could be having a non-email filled in, for example. this is why it's great :)

the reason I say is because of the naming of the const.